packages/core/src/tools/experimentalFeatures.ts

```diff
@@ -22,7 +22,7 @@ export enum ExperimentalFeature {
 
 const enabledExperimentalFeatures: Set<ExperimentalFeature> = new Set()
 
-export function initFeatureFlags(enableExperimentalFeatures?: string[] | undefined) {
+export function initFeatureFlags(enableExperimentalFeatures: string[] | undefined) {
```

Comment:
<!-- Praises highlight something positive. Try to leave at least one of these comments per review (if it exists :^)  -->
**👏 praise:** ‏ Nice!